Kryptonite/AppDelegate.swift

```diff
@@ -197,6 +204,14 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         handleAction(userInfo: notification.userInfo, identifier: identifier, completionHandler: completionHandler)
         
     }
+    
+    // UNUserNotificationCenterDelegate
+    @available(iOS 10.0, *)
+    func userNotificationCenter(_ center: UNUserNotificationCenter, didReceive response: UNNotificationResponse,withCompletionHandler completionHandler: @escaping () -> Void) {
```

Comment:
I'm not sure if this is strictly necessary, just given the inconsistency with the notifications framework  I want to make sure every delegate is implemented